Ansible/roles/django/tasks/main.yml

```diff
@@ -0,0 +1,59 @@
+- name: Clone git repo
+  ansible.builtin.git:
+    repo: "https://github.com/yeshwanthlm/django-on-ec2.git"
+    dest: "{{ project_dir }}"
+    force: yes
+
+- name: Update apt
+  apt:
+    update_cache: yes
+    upgrade: dist
+
+- name: Install python3-venv
+  apt:
+    name: python3-venv
+    state: present
+
+- name: Change permissions of manage.py
+  file:
+    path: "{{ project_dir }}/manage.py"
+    mode: '0755'
+
+- name: Create a virtual environment inside the project directory
```

Comment:
please use module for this

Reply:
Ok. Used pip module for creating virtual environment.